Makefile

```diff
@@ -502,8 +502,16 @@ test-vault:
 	@# Dont cache these results as they require the pachd cluster
 	go test -v -count 1 ./src/plugin/vault -timeout $(TIMEOUT)
 
-test-s3:
-	go test -v ./src/server/pfs/s3 -timeout $(TIMEOUT)
+./etc/testing/s3gateway/s3-tests:
+	cd ./etc/testing/s3gateway && git clone git@github.com:ceph/s3-tests.git
+	cd ./etc/testing/s3gateway/s3-tests && ./bootstrap
+	cd ./etc/testing/s3gateway.s3-tests && source virtualenv/bin/activate && pip install nose-exclude==0.5.0
+
+test-s3gateway-integration:
+	go test -v ./src/server/pfs/s3 -timeout $(TIMEOUT) -count 1 | grep -v 'INFO pfs.'
+
+test-s3gateway-conformance: ./etc/testing/s3gateway/s3-tests install
+	./etc/testing/s3gateway/conformance.py
```

Comment:
Something weird is happening with the diff here, it seems like I'm seeing s3 stuff, which was just merged into master... not quite sure what's going on.

Reply:
Whoops - I merged `master` this morning but messed up my state by stashing before the commit so it didn't realize it was a merge commit.  Should be fixed now.